src/components/CustomButton.tsx

```diff
@@ -1,49 +1,55 @@
 import React from 'react';
 import { StyleSheet, Pressable, Text, GestureResponderEvent } from 'react-native';
 
+import { swDarkBlue, swWhite, swPaleViolet } from '../constants/colors';
+
 interface ICustomButton {
-    // eslint-disable-next-line no-unused-vars
-    onPress: (event: GestureResponderEvent) => void,
-    title: string,
-    style?: string,
+  // eslint-disable-next-line no-unused-vars
+  onPress: (event: GestureResponderEvent) => void;
+  title: string;
+  type?: string;
```

Comment:
Potrzebne przeladowywanie styli

Reply:
Nadal nie wiem jak to zrobić, żeby TS nie krzyczał...